apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js

```diff
@@ -0,0 +1,85 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { ApplicationsHomePage } from '../../page_objects/applicationsHomePage';
+import { CreateCasePage } from '../../page_objects/createCasePage';
+import { SearchResultsPage } from '../../page_objects/searchResultsPage';
+import { ProjectTeamPage } from '../../page_objects/projectTeamPage';
+import { projectInformation } from '../../support/utils/createProjectInformation';
+
+const createCasePage = new CreateCasePage();
+const projectTeamPage = new ProjectTeamPage();
+const applicationsHomePage = new ApplicationsHomePage();
+const searchResultsPage = new SearchResultsPage();
+const { applications: applicationUsers } = users;
+
+describe('Create case ', () => {
+	let projectInfo;
+
+	before(() => {
+		projectInfo = projectInformation();
+		cy.login(applicationUsers.caseAdmin);
+		createCasePage.createCase(projectInfo);
+	});
+
+	it('Add team member and verify the role is added to project team', () => {
+		cy.login(applicationUsers.caseAdmin);
+		cy.visit('/');
+		const caseRef = Cypress.env('currentCreatedCase');
+		applicationsHomePage.searchFor(caseRef);
+		searchResultsPage.clickTopSearchResult();
+		searchResultsPage.clickLinkByText('Project team');
+		searchResultsPage.clickButtonByText('Add team member');
+		projectTeamPage.addTeamMeber('caseofficeradmin.test@planninginspectorate.gov.uk');
+		projectTeamPage.verifyCaseManagerRoleAdded();
+	});
+
+	it('Verify team member is added', () => {
+		cy.login(applicationUsers.caseAdmin);
+		cy.visit('/');
+		const caseRef = Cypress.env('currentCreatedCase');
+		applicationsHomePage.searchFor(caseRef);
+		searchResultsPage.clickTopSearchResult();
+		searchResultsPage.clickLinkByText('Project team');
+		searchResultsPage.clickButtonByText('Add team member');
+		projectTeamPage.searchTeamMemberByEmail('caseofficeradmin.test@planninginspectorate.gov.uk');
```

Comment:
when/if `caseofficeradmin.test@planninginspectorate.gov.uk` is decommissioned this test will not return results

Reply:
caseofficer.test@planninginspectorate.gov.uk is decommissioned not caseofficeradmin.test@planninginspectorate.gov.uk

Reply:
sure